src/core/etl_phases/event_enhancers.py

```
@@ -159,13 +162,13 @@ def find_shift_id(row):
-    shot_type = pd.read_csv(output_dir / 'dim_shot_type.csv', low_memory=False)
-    ze_type = pd.read_csv(output_dir / 'dim_zone_entry_type.csv', low_memory=False)
-    zx_type = pd.read_csv(output_dir / 'dim_zone_exit_type.csv', low_memory=False)
-    pass_type = pd.read_csv(output_dir / 'dim_pass_type.csv', low_memory=False)
-    stoppage_type = pd.read_csv(output_dir / 'dim_stoppage_type.csv', low_memory=False)
-    giveaway_type = pd.read_csv(output_dir / 'dim_giveaway_type.csv', low_memory=False)
-    takeaway_type = pd.read_csv(output_dir / 'dim_takeaway_type.csv', low_memory=False)
+    shot_type = safe_read_csv(output_dir / 'dim_shot_type.csv')
+    ze_type = safe_read_csv(output_dir / 'dim_zone_entry_type.csv')
+    zx_type = safe_read_csv(output_dir / 'dim_zone_exit_type.csv')
+    pass_type = safe_read_csv(output_dir / 'dim_pass_type.csv')
+    stoppage_type = safe_read_csv(output_dir / 'dim_stoppage_type.csv')
```

Ruff (0.14.13)

Local variable `stoppage_type` is assigned to but never used

Remove assignment to unused variable `stoppage_type`

(F841)